Tasks/DockerComposeV0/dockercomposeup.ts

```diff
@@ -26,5 +27,13 @@ export function run(connection: DockerComposeConnection): any {
     var arg = tl.getInput("arguments", false);
     var commandArgs = dockerCommandUtils.getCommandArguments(arg || "");
     command.line(commandArgs || "");
-    return connection.execCommand(command);
+
+    // setup variable to store the command output
+    let output = "";
+    command.on("stdout", data => {
+        output += data;
+    });
+
+    return connection.execCommand(command)
+    .then(() => outputUpdate(utils.writeTaskOutput("up", output)))
```

Comment:
Instead, you could make all the calls `execSync` and use stdout value. 
This change seems to introduce lot of duplicate logic

Reply:
@ds-ms, could you elaborate on this?

Do you mean that I should create a new function on the dockerComposeConnection named execSync which returns the output of the command as a Promise<string>?

Reply:
My mistake, I wasn't aware of this wrapper function. You can let this be. 

What I meant was, the `azure-pipelines-task-lib` module offers two ways of running commands.  For async you use `exec()` and for sync you use `execSync()`. This `execCommand` seems to be a wrapper function over `exec()`. It's okay to keep the entire logic `async` here.

Reply:
Do you still want me to create a new function inside the connection class then to try to reduce duplication?

Reply:
That shouldn't be necessary